src/js/enableAirPlay.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
Could we add these 'use strict' statements back to the files they were in? It's our standard to include them in JS files. Thanks!

Reply:
Sure, sorry about that.  I don't know why I did that :man_shrugging: